scripts/one_roll_engine.js

```diff
@@ -112,19 +121,27 @@ const createRawRoll = (diceCount) => {
  * @param {string} flavorText - e.g. "Flaming sword attack"
  * @returns {ORERollResult}
  */
-const parseRawRoll = (roll, flavorText) => {
+const parseRawRoll = (roll, expertCount, expertValue, flavorText) => {
   const rawRolls = roll.terms[0].results.map(r => r.result)
+  const expertRolls = new Roll(`${expertCount}d${expertValue}`).roll({ async: false, maximize: true }).terms[0].results.map(r => r.result)
   const counts = new Array(11).fill(0)  // [0, 1, ..., 9, 10].  the 0 is not used
   rawRolls.forEach(k => {
     counts[k] += 1
   })
+  expertRolls.forEach(k => {
+    counts[k] += 1
+  })
   const sets = {}  // key = height, value = width
   const looseDice = []
   counts.forEach((count, num) => {
     if (count === 0) return  // (will also skip the "0" count)
     if (count === 1) looseDice.push(num)
     if (count >= 2) sets[num] = count
   })
+  /*const masterDice = []
+  for (let i = 0; i < masterCount; i++) {
+    masterDice.push(10) 
+  }*/
```

Comment:
leftover comment?

Reply:
Yep, I was playing around with ways of doing master/wiggle dice and forgot to take that out.